vendor/modules.txt

```diff
@@ -475,7 +475,6 @@ github.com/opencontainers/image-spec/specs-go
 github.com/opencontainers/image-spec/specs-go/v1
```

Comment:
Given that we are only using the `kafka/proto` we can remove the replace directive for this dependency in `go.mod` and use the one from upstream instead our fork?

Reply:
I was not aware of the replace directive, will check!

Reply:
@aanm The three commits we have on our fork a head of the upstream ancestor are all on `kafka/proto`. Now that we are not parsing requests at all some of there commits may not be needed any more, but the latest commit fixes an issue in request parsing that seems like we'd want to keep.